includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```
@@ -2149,6 +2152,201 @@ public function process_upe_redirect_payment( $order_id, $intent_id, $save_payme
+	 * @param bool $is_pay_for_order True when the customer is on the pay-for-order endpoint.
+	 *
+	 * @since 10.6.0
+	 */
+	public function process_checkout_session_redirect( int $order_id, bool $is_pay_for_order = false ): void {
+		$order = wc_get_order( $order_id );
+
+		if ( ! $order instanceof WC_Order ) {
+			return;
+		}
+
+		$order_helper = WC_Stripe_Order_Helper::get_instance();
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Order_Helper' in method 'process_checkout_session_redirect'. (undefined)

(StaticAccess)